pkg/tracing/tracing.go

```diff
@@ -13,58 +18,269 @@ import (
 
 // Commandline options for exporter endpoints
 const (
-	jaegerEndpointOpt      = "jaeger-endpoint"
-	jaegerAgentEndpointOpt = "jaeger-agent-endpoint"
+	jaegerEndpointOpt       = "jaeger-endpoint"
+	jaegerAgentEndpointOpt  = "jaeger-agent-endpoint"
+	jaegerSamplerOpt        = "jaeger-sampler"
+	jaegerSampleFractionOpt = "jaeger-sample-fraction"
+)
+
+// JaegerSamplerType to indicate different Jaeger sampler type
+type JaegerSamplerType uint8
+
+// Sampler types in Jaeger
+const (
+	// 'Never' - Don't sample any trace
+	Never JaegerSamplerType = iota
+	// 'Always' - Sample every trace
+	Always
+	// 'Probabilistic' - Sample based on sample fraction
+	Probabilistic
 )
 
+// SamplerTypeToString returns string representation of sampler type.
+func SamplerTypeToString(samplerType JaegerSamplerType) string {
+	switch samplerType {
```

Comment:
same here, this can be simplified as suggested in one of the comment, 

can we call this function for https://github.com/gluster/glusterd2/pull/1480/files#diff-8918aaf56f805e88174f0f9e8599772bR99

Reply:
There's only one function now under pkg/tracing/tracing.go and that is being used.